scio-extra/src/main/scala/com/spotify/scio/extra/libsvm/package.scala

```diff
@@ -0,0 +1,80 @@
+package com.spotify.scio.extra
```

Comment:
Missing license header
See this doc: https://github.com/spotify/scio/wiki/Style-Guide